js/mqtt/mqtt-websocket.js

```diff
@@ -1,8 +1,9 @@
 class MQTTWebSocket {
 
-  constructor(address, username, password, onConnect) {
+  constructor(address, username, password) {
```

Comment:
Please put the `onConnect` back and call it in the lambda function that you added. We need to call the handler's `success` method when MQTT connects

Reply:
There's only one issue with `onConnect` at line 24. It should be `onConnect();` not `OnConnect;`. I'll fix that after merging.